tide_landing_page.module

```diff
@@ -71,5 +71,10 @@ function tide_landing_page_form_node_form_alter(&$form, FormStateInterface $form
   if (in_array($form_id, ['node_landing_page_form', 'node_landing_page_edit_form'])) {
     // Change form layout.
     $form['#attached']['library'][] = 'tide_landing_page/landing_page_form';
+
+    // Hide the field Show Acknowledgement to Country if tide_site is missing.
+    if (!\Drupal::moduleHandler()->moduleExists('tide_site')) {
```

Comment:
@sonnykt Why is this required?

Reply:
@GROwen The "Acknowledgement to Country" field in the Site term is provided by Tide Site. Without Tide Site, the "Show Acknowledgement to Country?" becomes useless and IMO it should be hidden to avoid any confusion.